src/main/java/controller/StudentsList.java

```diff
@@ -0,0 +1,35 @@
+package controller;
+
+import model.Student;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by aleksey.dobrovolsky on 6/3/2019.
+ */
+class StudentsList {
+
+    static List<Student> students = init();
+
+    private static List<Student> init() {
+        List<Student> students = new ArrayList<Student>();
+        String[] firstNames = {"Olivia", "Amelia", "Isla", "Emily", "Ava", "Oliver", "Harry", "Jack", "George", "Noah"};
+        String[] lastNames = {"Smith", "Johnson", "Williams", "Jones", "Brown", "Davis", "Miller", "Wilson", "Moore", "Taylor"};
+        int[] ages = {18, 20, 21, 22, 18, 18, 19, 23, 23, 20};
+        int[] ids = {96, 97, 98, 99, 100, 101, 102, 103, 104, 105};
+
+        for (int i = 0; i < 10; i++) {
+            students.add(new Student(firstNames[i], lastNames[i], ids[i], ages[i]));
+        }
+        return students;
+    }
+
+    static void resultList(List<Student> list) {
```

Comment:
this method doesn't belong to the StudentList class. Yes, it prints the list of students but only those that are passed as argument. To make method linked to the StudentList class, it should print the list of students that is property of the class, ie. "students" list.

Reply:
done - print is not static right now and works with the list of attributes related to the object